avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -365,7 +364,7 @@ entShearResistance = 0
 entDefResistance = 0
```

Comment:
These need units and explanations as well, no?

Reply:
Mh, leave these as is, they are in the code but might get removed. Add above entShearResistance:
Used to determine speed loss via energy loss due to entrained mass